GNUmakefile

```diff
@@ -2,13 +2,11 @@
 # https://www.consul.io/docs/install#compiling-from-source
 
 SHELL = bash
-GOGOVERSION?=$(shell grep github.com/gogo/protobuf go.mod | awk '{print $$2}')
 GOTOOLS = \
 	github.com/elazarl/go-bindata-assetfs/go-bindata-assetfs@master \
```

Comment:
This section is missing the `github.com/golang/protobuf/protoc-gen-go@v1.3.5` install (and also should get version intepolation as above).